aliases.sh

```diff
@@ -16,3 +16,16 @@ alias git-global-ignore='git config --global core.excludesfile "~/utils/gitignor
 alias sf-cs-fix='php-cs-fixer fix --rules=@Symfony';
 
 alias gz='gzip --keep';
+
+if [ 'Darwin' = `uname` ]; then
+  # Mac OS X
+  alias lsd='ls -GhalF@';
+else
+  alias lsd='ls -halF --color';
+fi;
+alias grep='grep --color=always';
```

Comment:
`ls -GhalF@ | grep --color=always rc | less --RAW-CONTROL-CHARS`

* `grep` doesn't care that `ls` was colorized.
* `grep` need `always` to pipe colors (as `auto` only colorized when not piped).

If piped to a command not handling color control characters, `grep --color=always` will have an ugly, almost useless, output.

```suggestion
alias grep='grep --color=auto';
alias grepc='grep --color=always';
```

Reply:
Last `--color` option wins. 
* `ls -halF | grep --color=auto --color=always rc | less -R`: output is colorized
* `ls -halF | grep --color=always --color=auto rc | less -R`: output isn't colorized

In the following on a terminal, both outputs are colorized.

```shell
alias grep='grep --color=auto';
ls -halF | grep rc;
ls -halF | grep --color=always rc | less -R;
```